applications/teletype/src/teletype_util.erl

```diff
@@ -930,35 +930,50 @@ find_addresses(DataJObj, TemplateMetaJObj, ConfigCat) ->
     AddressKeys = [<<"to">>, <<"cc">>, <<"bcc">>, <<"from">>, <<"reply_to">>],
     find_addresses(DataJObj, TemplateMetaJObj, ConfigCat, AddressKeys, []).
 
-find_addresses(_DataJObj, _TemplateJObj, _ConfigCat, [], Acc) -> Acc;
+find_addresses(_, _, _, [], Acc) -> Acc;
```

Comment:
`_DataJObj` and friends are convenient documetation for what those args represent; why remove them?